src/main.rs

```diff
@@ -79,6 +79,8 @@ async fn main() -> Result<(), anyhow::Error> {
             };
 
             keyturner.run_action(action).await?;
+
+            stream.forward(futures::sink::drain());
```

Comment:
das würde auch gehen
```suggestion
            while futures_util::poll!(stream.next()).is_ready() {}
```